### PR TITLE
[Feature] Screen adaptiveness for large screens

### PR DESCRIPTION
**Pull request by Mrayyan**
## Purpose

Composite is now screen adaptive for large screens (7inch +). Things changed:
> Call grid now goes up to 9 remote participants
> Call banner shorten width + cantered
> Setup screen changed on tablets
> Avatar view inside pip
> PIP size on tablets

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Open...
* Click on...

## What to Check
Verify that the following are valid
Please test all of these things:
> Call grid now goes up to 9 remote participants
> Call banner shorten width + cantered
> Setup screen changed on tablets
> Avatar view inside pip
> PIP size on tablets

Make sure they have changed and look good.

### CODE REVIEW

File: azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/di/DependencyInjectionContainerImpl.kt [hunk 1]
```diff
@@ -98,11 +99,14 @@ internal class DependencyInjectionContainerImpl(
     }
 
     override val appStore by lazy {
+        val maxRemoteParticipantLimit = applicationContext.resources.getInteger(R.integer.azure_communication_ui_calling_max_remote_participants)
+
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/di/DependencyInjectionContainerImpl.kt:103]: App store is related to redux state, action management. We should not inject anything related to particiapnts in App Store.

File: azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/redux/Store.kt [hunk 2]
```diff
@@ -11,4 +11,5 @@ internal interface Store<S> {
     fun getStateFlow(): MutableStateFlow<S>
     fun getCurrentState(): S
     fun end()
+    fun getMaxRemoteParticipantLimit(): Int
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/redux/Store.kt:14]: This part can be included in view layer.

File: azure-communication-ui/azure-communication-ui/src/calling/res/layout/azure_communication_ui_calling_avatar_view_in_pip.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui/src/calling/res/layout/azure_communication_ui_calling_avatar_view_in_pip.xml:1]: missing copyright

File: azure-communication-ui/azure-communication-ui/src/calling/res/layout-land/azure_communication_ui_calling_avatar_view_in_pip.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui/src/calling/res/layout-land/azure_communication_ui_calling_avatar_view_in_pip.xml:1]: missing copy right

File: azure-communication-ui/azure-communication-ui/src/calling/res/layout-sw600dp/azure_communication_ui_calling_avatar_view_in_pip.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui/src/calling/res/layout-sw600dp/azure_communication_ui_calling_avatar_view_in_pip.xml:1]: missing copyright

File: azure-communication-ui/azure-communication-ui/src/calling/res/values/azure_communication_ui_calling_integers.xml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui/src/calling/res/values/azure_communication_ui_calling_integers.xml:1]: copyright

**Review by iaulakh — CHANGES_REQUESTED** [review]

**Review by iaulakh — APPROVED** [review]

**Review by phileo99 — APPROVED** [review]
Tested with 9, 3 and 2 participants, grid resizes as expected.
There was an issue with the participant banner, but this will be dealt with in a separate PR

**Pull request merged**